hedera-node/hedera-evm/src/test/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmCreateOperationTest.java

```diff
@@ -71,10 +75,29 @@ void computesExpectedCost() {
 
     @Test
     void computesExpectedTargetAddress() {
-        given(frame.getWorldUpdater()).willReturn(hederaWorldUpdater);
+        given(frame.getWorldUpdater()).willReturn(stackedUpdater);
         given(frame.getRecipientAddress()).willReturn(recipientAddr);
-        given(hederaWorldUpdater.newContractAddress(recipientAddr)).willReturn(Address.ZERO);
+        given(stackedUpdater.get(recipientAddr)).willReturn(sender);
+        given(sender.getNonce()).willReturn(1L);
```

Comment:
I know it gets hashed, so in some sense any number would do, but it seems a good idea (to me) to avoid `1L` here as combined with the `-1` it's going to end up as `0` and using either `1` or `0` in this kind of test when you have free choice of any number seems like you might run into uses of "defaults' or something where what you think is being tested is not what is being tested.  Do you know what I mean?  In general, anyway, not necessarily in this particular case, where, like I said, I know it's going into a hash.